entity-framework/core/what-is-new/ef-core-10.0/whatsnew.md

```diff
@@ -110,3 +122,4 @@ Thanks to [@aradalvand](https://github.com/aradalvand) for proposing and pushing
 
 * Make SQL Server scaffolding compatible with Azure Data Explorer ([#34832](https://github.com/dotnet/efcore/pull/34832), contributed by [@barnuri](https://github.com/barnuri)).
 * Associate the DatabaseRoot with the scoped options instance and not the singleton options ([#34477](https://github.com/dotnet/efcore/pull/34477), contributed by [@koenigst](https://github.com/koenigst)).
+* Redact inlined constants from log when sensitive logging is off ([#35724](https://github.com/dotnet/efcore/pull/35724)).
```

Comment:
...or should we have a separate paragraph for this? @cincuranet

Reply:
I agree. Not worth wasting the Internet bytes for this.